man/tpm2_takeownership.1.md

```diff
@@ -4,8 +4,8 @@
 
 # NAME
 
-**tpm2_takeownership**(1) - Insert authorization values for the owner, endorsement
-and lockout authorizations.
+**tpm2_takeownership**(1) - Insert authorization values for the owner, platform,
```

Comment:
By programming platform auth, this tool is definitely doing one more than take_ownership in strict sense. As for this commit, this looks ok.